controllers/articles-controller.js

```diff
@@ -0,0 +1,13 @@
+const { fetchArticleById } = require('../models/articles-model')
+
+const getArticleById = (request, response, next) => {
+    const articleId = Number(request.params.article_id)
+    fetchArticleById(articleId).then((article) => {
+        response.status(200).send({article})
+    })
+    .catch((err) => {
+        next(err)
+    })
+}
+
```

Comment:
- Well named function name `getArticleById` - well done
- Just a tip you don't need to coerce the id to a number yourself, psql will do this for you
- The response being sent on a key of `article` is spot on and appropriate, well done

Reply:
Update: I can see you're doing some form of checking in the model on if its a number, in which case: this is fine